admin/src/js/services/db.js

```diff
@@ -22,6 +22,7 @@ angular.module('inboxServices').factory('DB',
     const isOnlineOnly = Session.isOnlineOnly();
 
     const getUsername = remote => {
+      Session.checkCurrentSession();
```

Comment:
This gets called immediately when coming back to the admin app with the back button. It strangely gets called before the `pageshow` event handler and this function throwing an error results in the event handler not being called 🤨

But hey, no need for an extra event listener at least :)